ip6addr.js

```diff
@@ -245,7 +245,7 @@ Addr.prototype.toBuffer = function toBuffer(buf) {
       throw new Error('optional arg must be Buffer');
     }
   } else {
-    buf = new Buffer(16);
+    buf = Buffer.alloc(16);
```

Comment:
I'd prefer to add backwards compat here (to support earlier than node v5.10.0), i.e.:
```suggestion
    if (Buffer.alloc) {
        buf = Buffer.alloc(16);
    } else {
        buf = new Buffer(16);
    }
```

Reply:
This seems like a good idea to me